### PR TITLE
2.x moving to jakarta

### PR DESCRIPTION
**Pull request by amoscatelli**
Hi @t1 !

If you are intested, I migrated the jax-rs and lib to jakarta.

### CODE REVIEW

File: jaxrs/pom.xml [hunk 1]
```diff
@@ -13,6 +13,11 @@
 
     <artifactId>problem-details-jaxrs</artifactId>
     <description>An implementation for JAX-RS using an ExceptionMapper</description>
+    
+    <properties>
```
Comment (t1) [inline jaxrs/pom.xml:17]: Shouldn't the properties defined in the parent pom be adjusted instead?

File: jaxrs/pom.xml [hunk 2]
```diff
@@ -64,14 +69,20 @@
         <dependency>
             <groupId>org.eclipse.microprofile</groupId>
             <artifactId>microprofile</artifactId>
-            <version>${microprofile.version}</version>
+            <version>5.0</version>
```
Comment (t1) [inline jaxrs/pom.xml:72]: Why not use the property any more?

File: jaxrs/pom.xml [hunk 3]
```diff
@@ -128,6 +139,12 @@
             <version>${yasson.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (t1) [inline jaxrs/pom.xml:142]: Just asking: Why do we need geronimo?

**Review by t1 — CHANGES_REQUESTED** [review]
Great idea and even bigger thanks for the help! I have a few questions, though.

**Comment by t1** [comment]
BTW, @amoscatelli: You are aware that the 2.x branch has not been released to maven central, yet?!?

**Comment by amoscatelli** [comment]
Yea, but I need jax-rs support so I compiled it myself 2 years ago and I am using it since then.

File: jaxrs/pom.xml (hunk 3 above)
Comment (amoscatelli) [reply jaxrs/pom.xml:142]: We need a microprofile config provider to pass the test.
I suppose this is required by the new resteasy ...

File: jaxrs/pom.xml (hunk 2 above)
Comment (amoscatelli) [reply jaxrs/pom.xml:72]: You are right, going to use the property, I forgot to restore it

File: jaxrs/pom.xml (hunk 1 above)
Comment (amoscatelli) [reply jaxrs/pom.xml:17]: That's ok for me, but then even the spring version would be impacted.

Comment (t1) [reply jaxrs/pom.xml:17]: But both properties are not used by the spring modules

**Comment by t1** [comment]
And that was the 2.x branch?

**Comment by t1** [comment]
I renamed the branches, so it's clear to see what they are for. And I just released version 1.0.10.
No I will upgrade `javax→jakarta`, as you've suggested, but based on the former 1.x branch... and then release a 2.0.0.

**Comment by t1** [comment]
I just released 2.0.0. It will take a while to sync to maven central. Thanks for triggering me!

**Comment by amoscatelli** [comment]
Sorry I couln't answer during the weekend, I feel like I missed something lol

**Pull request closed without merging**